keras_cv/models/object_detection/faster_rcnn/faster_rcnn_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv.backend import keras
+from keras_cv.models import ResNet18V2Backbone
+from keras_cv.models.object_detection.__test_utils__ import (
+    _create_bounding_box_dataset,
+)
+from keras_cv.models.object_detection.faster_rcnn.faster_rcnn import FasterRCNN
+from keras_cv.tests.test_case import TestCase
+
+
+class FasterRCNNTest(TestCase):
```

Comment:
This is great -- let's also add equivalents to these YOLOV8 tests:

test_saved_model
test_backbone_preset
test_fit_with_no_valid_gt_bbox